core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/factory.go

```diff
@@ -94,8 +98,6 @@ func (o *LogTriggersOptions) defaultBlockRate() uint32 {
 
 func (o *LogTriggersOptions) defaultLogLimit() uint32 {
 	switch o.chainID.Int64() {
-	case 42161, 421613, 421614: // Arbitrum
-		return 1
```

Comment:
discussed offline: change default to 1 and arbitrum block rate to 2 in above function